scripts/nginx/readarr.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# Nginx conf for Readarr
+# Bakerboy448 for swizzin 2020
+# Based on Radarr v3 installationn by flying sausages
+master=$(cut -d: -f1 < /root/.master.info)
+
+cat > /etc/nginx/apps/readarr.conf << readarr
+location /readarr {
+  proxy_pass        http://127.0.0.1:8787/readarr;
```

Comment:
I think these ports will conflict, I'll double check later

Reply:
The ports **WILL** conflict with radarr

Reply:
radarr is 7878
readarr is 8787 
no?

Reply:
Looks like it's the SSL port. People that choose not to run panel are very likely to use this.
<img width="318" alt="image" src="https://user-images.githubusercontent.com/23618693/102721337-79580180-42fa-11eb-8188-c4d24bb30519.png">

Reply:
updated the nginx config for radarr;  what's the method to check if radarr ssl is set to 8787 and update everything accordingly  (picked 7879 for the change to nginx/radarr)?

Reply:
non-issue upstream 9898 is default in v3
https://github.com/Radarr/Radarr/blob/develop/src/NzbDrone.Core/Configuration/ConfigFileProvider.cs#L140

Reply:
Yeah that's good to know. We'll have to discuss what the best way to do that internally is but that will be an updater script